package.json

```diff
@@ -137,5 +138,27 @@
     "sinon": "7.2.2",
     "url-polyfill": "1.0.9",
     "web-push-notifications": "3.4.7"
+  },
+  "nyc": {
+    "reporter": [
+      "text-summary",
+      "lcov"
+    ],
+    "temp-directory": "./coverage/.nyc_output",
+    "all": true,
+    "exclude": "typings",
+    "extension": [
+      ".js",
+      ".ts",
```

Comment:
We don't need `.ts` and `.tsx` as they are not supported yet in binary.

Reply:
sure, it can be added whenever we migrate to typescript